code/modules/antagonists/nightmare/nightmare_equipment.dm

```diff
@@ -10,7 +10,8 @@
 	armour_penetration = 35
 	lefthand_file = 'icons/mob/inhands/antag/changeling_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/antag/changeling_righthand.dmi'
-	item_flags = ABSTRACT | DROPDEL | ACID_PROOF
+	item_flags = ABSTRACT | DROPDEL
+	resistance_flags = INDESTRUCTIBLE | ACID_PROOF
```

Comment:
if its indestructible doesnt that mean its acid proof by default?

Reply:
fun fact, no it doesnt
confirmed by looking through code, almost everything that uses INDESTRUCTIBLE uses like 4 other flags, those being:
FIRE_PROOF
LAVA_PROOF
UNACIDABLE
ACID_PROOF

in light of this i added those to the resistance_flags, should make the visuals not appear on the item either and no weirdness should happen

more of a precaution, no?

Reply:
yep went through them again, almost all of them have those 5 flags